packages/babel-core/src/transformation/normalize-file.js

```diff
@@ -110,6 +109,7 @@ function parser(pluginPasses, options, code) {
         err.message =
           `${options.filename || "unknown"}: ${err.message}\n\n` + codeFrame;
       }
+      err.code = "BABEL_PARSE_ERROR";
```

Comment:
I don't know if this is the first time we started doing this but are we going to have some kind of convention for this, docs? Or is this just a one-off @loganfsmyth

Reply:
I think error codes are good. Probably in the long run anywhere we throw an error in babel-core should have its own code. Even better would be if Babylon already included a code and this code checked
```
if (err.code === "BABYLON_PARSE_ERROR")
```

Reply:
Ok yeah ^ was what I was thinking. maybe @kaicataldo can you make an issue about that? I think a task could be to look through all the places we throw and to think about if we want to make an errors file kind of deal

Reply:
Would be great to make this consistent! Would this be just for Babylon and babel-core? Or all packages?

Reply:
It's funny also because I'm reminded we have a package called `babel-messages` that we sometimes use?